src/app/register-form/register-form.component.html

```diff
@@ -0,0 +1,190 @@
+<div class="container">
+  <form [formGroup]="registerForm" (ngSubmit)="onSubmit(); registerForm.reset()">
```

Comment:
Best to move the two calls into a separate function in typescript and call everything there. Will be magical once you wanna test it.